<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Carotid Analyzer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/resources/branch.svg</normaloff>:/icons/resources/branch.svg</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="central_widget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="central_widget_layout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="module_stack">
      <widget class="QWidget" name="empty_module">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="empty_module_label">
          <property name="text">
           <string>Select a module.</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_load_new_DICOM"/>
    <addaction name="action_set_working_directory"/>
    <addaction name="action_delete_selected_patient"/>
    <addaction name="separator"/>
    <addaction name="action_save_and_propagate"/>
    <addaction name="action_discard_changes"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="action_data_inspector"/>
    <addaction name="separator"/>
    <addaction name="action_crop_module"/>
    <addaction name="action_segmentation_module"/>
    <addaction name="action_centerline_module"/>
    <addaction name="separator"/>
    <addaction name="action_stenosis_classifier"/>
    <addaction name="action_flowComp_module"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolbar_modules">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_data_inspector"/>
   <addaction name="separator"/>
   <addaction name="action_crop_module"/>
   <addaction name="action_segmentation_module"/>
   <addaction name="action_centerline_module"/>
   <addaction name="separator"/>
   <addaction name="action_stenosis_classifier"/>
   <addaction name="action_flowComp_module"/>
  </widget>
  <widget class="QToolBar" name="toolbar_save">
   <property name="windowTitle">
    <string>toolbar_save</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_save_and_propagate"/>
   <addaction name="action_discard_changes"/>
  </widget>
  <widget class="QDockWidget" name="dock_data_inspector">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>524287</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>300</width>
     <height>0</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Data Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="data_inspector_contents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QTreeWidget" name="tree_widget_data">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="indentation">
        <number>20</number>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
       <property name="allColumnsShowFocus">
        <bool>true</bool>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <attribute name="headerDefaultSectionSize">
        <number>90</number>
       </attribute>
       <attribute name="headerHighlightSections">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Patient ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Left</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Right</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_load_file">
       <property name="text">
        <string>Load Selected Patient</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_load_new_DICOM">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/database-plus.svg</normaloff>:/icons/resources/database-plus.svg</iconset>
   </property>
   <property name="text">
    <string>Load New DICOM...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_set_working_directory">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/folder-account.svg</normaloff>:/icons/resources/folder-account.svg</iconset>
   </property>
   <property name="text">
    <string>Set Working Directory...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_save_and_propagate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/progress-check.svg</normaloff>:/icons/resources/progress-check.svg</iconset>
   </property>
   <property name="text">
    <string>Save And Propagate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_crop_module">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/crop.svg</normaloff>:/icons/resources/crop.svg</iconset>
   </property>
   <property name="text">
    <string>Crop Module</string>
   </property>
  </action>
  <action name="action_segmentation_module">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/drawing-box.svg</normaloff>:/icons/resources/drawing-box.svg</iconset>
   </property>
   <property name="text">
    <string>Segmentation Module</string>
   </property>
  </action>
  <action name="action_centerline_module">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/vector-polyline-edit.svg</normaloff>:/icons/resources/vector-polyline-edit.svg</iconset>
   </property>
   <property name="text">
    <string>Centerline Module</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="action_stenosis_classifier">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/chart-bell-curve-cumulative.svg</normaloff>:/icons/resources/chart-bell-curve-cumulative.svg</iconset>
   </property>
   <property name="text">
    <string>Stenosis Classifier</string>
   </property>
  </action>
  <action name="action_data_inspector">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/database-eye.svg</normaloff>:/icons/resources/database-eye.svg</iconset>
   </property>
   <property name="text">
    <string>Data Inspector</string>
   </property>
  </action>
  <action name="action_discard_changes">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/delete-forever.svg</normaloff>:/icons/resources/delete-forever.svg</iconset>
   </property>
   <property name="text">
    <string>Discard Changes</string>
   </property>
  </action>
  <action name="action_flowComp_module">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/compare.svg</normaloff>:/icons/resources/compare.svg</iconset>
   </property>
   <property name="text">
    <string>FlowComp Module</string>
   </property>
  </action>
  <action name="action_delete_selected_patient">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/delete-remove.svg</normaloff>:/icons/resources/delete-remove.svg</iconset>
   </property>
   <property name="text">
    <string>Delete Selected Case</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
